core/libp2p/connection/yamux/yamux_frame.cpp

```diff
@@ -28,10 +28,11 @@ namespace {
   }
 }  // namespace
 
-namespace libp2p::muxer {
-  kagome::common::Buffer YamuxFrame::frameBytes(
-      uint8_t version, FrameType type, Flag flag, uint32_t stream_id,
-      uint32_t length, const kagome::common::Buffer &data) {
+namespace libp2p::connection {
+  kagome::common::Buffer YamuxFrame::frameBytes(uint8_t version, FrameType type,
```

Comment:
Function with that big number of arguments is generally no good. Is it possible to refactor it so that it would have 2-3 parameters? Perhaps create a separate type to store some.